_sass/spec/base.scss

```diff
@@ -199,6 +199,17 @@ $INFO_BORDER_COLOR: rgba(4, 66, 137, 0.2);
     }
   }
 
+  &-sitemap-external {
+    // Don't include a list item icon for external links.
+    display: inline-flex;
```

Comment:
Oh this is interesting, this prevents the marker from being added to the `<detail>` element. Do you know why it does that?

Reply:
I think the marker is only created if `display` is set to `list-item`. I got rid of the marker on purpose though: it says "drop down" to me, and there's nothing to drop down for an external link. If you prefer to have the marker for all of the items though, we could set `display` back to `list-item` and enclose the interior elements in a <div>.

Reply:
Oh Safari doesn't seem to display this behavior:
> <img width="470" alt="image" src="https://user-images.githubusercontent.com/12139762/179327002-4d134698-31e6-4102-8c87-78356c9d3044.png">

Fixed in https://github.com/eecs485staff/primer-spec/pull/195/commits/e32fb69dbad1794610a927e0bbd13a7c057916e2

Reply:
Woah, it's weird that Safari just ignores you if you try to say that it's not a list item!

Reply:
@noah-weingarden gotcha, and I agree with the rationale for getting rid of the marker. I was wondering why changing the `display` style even removes the marker. (It turns out that Safari is quirky and requires a slightly different solution.)